src/main/java/uk/gov/hmcts/reform/unspec/model/CaseData.java

```diff
@@ -88,4 +90,10 @@
     private final CloseClaim discontinueClaim;
 
     private final BusinessProcess businessProcess;
+
+    public boolean hasNoOngoingBusinessProcess() {
```

Comment:
Do we need to put `@JsonIgnore` for this? so when we serialize this using `CaseDetailsConverter` it's not sent to CCD.

Reply:
It's not required, Jackson uses `isPropertyName` to serialize boolean properties. And even if it wants to serialize it, we're still protected by our tests - they'd fail when callbacks would try to set non-existent case field.